doc/filtering.rst

```diff
@@ -96,7 +96,7 @@ This section contains functions to filter values during iteration.
    :returns: {gen1, param1, state1}, {gen2, param2, state2}
 
    The function returns two iterators where elements do and do not satisfy the
```

Comment:
A general comment: I would prefix all commits with `doc:` prefix, so it'll nicely look in git log.

Reply:
I have reviewed previous commits and all of them doesn't have prefixes, so I decided to write a similar commit header.
Ok, let's use prefixes.